freqtrade/rpc/telegram.py

```diff
@@ -23,6 +24,9 @@
 
 _UPDATER: Updater = None
 _CONF = {}
+_STAKE_CURRENCY = None
```

Comment:
I don't think `_STAKE_CURRENCY` and `_FIAT_CURRENCY` should be their own global variables, because they are already in `_CONF`. This could lead to inconsistencies. Please stick with `_CONF` for now until #36  is solved

Reply:
This is a great comment, I did not think about it. This is fixed